swap/src/alice.rs

```diff
@@ -433,8 +446,8 @@ impl Default for Alice {
 }
 
 fn calculate_amounts(btc: ::bitcoin::Amount) -> SwapAmounts {
-    // TODO: Get this from an exchange.
-    // This value corresponds to 100 XMR per BTC
+    // TODO (Franck): This should instead verify that the received amounts matches
```

Comment:
❓ I am not sure I understand this comment correctly. Is the goal to verify that the amount received (after the swap) matches the amount that was passed in?